### DISCUSSION TITLE
Questions about the prefab

### DISCUSSION THREAD

**Discussion by HannahHaensen** [julienkay/MobileNeRF-Unity-Viewer#5] [Q&A] [answered]
Dear Julien,

very nice work! 

I have a question about the available prefab, is this downloaded or also generated in your code?

Thanks in advance

**Comment by julienkay** [comment, marked as answer]
Hi Hannah,

yes the prefab is generated via code:
https://github.com/julienkay/MobileNeRF-Unity-Viewer/blob/b8a2ae24aaf8b3091adcd5cf05986fd8e5364029/Assets/Editor/MobileNeRFImporter.cs#L613-L627

The NeRFs are made up of several OBJs which are downloaded and the prefab combines all of them with the generated materials/shader already applied.

**Comment by HannahHaensen** [comment]
@julienkay 
ah thanks, probably overlooked during the search
